src/components/Button/README.md

```diff
@@ -504,9 +668,36 @@ export default {
 </script>
 
 <style scoped>
+.examples {
+	display: flex;
+	gap: 50px;
+	align-items: flex-start
```

Comment:
let's add `flex-wrap: wrap` here so that people don't have scroll sideways if they have a narrow viewport, the examples as-is are super wide (screenshot)

![image](https://user-images.githubusercontent.com/7769424/189106272-5f632e85-5086-4030-9eb5-64535f123d81.png)